transform/mattermost-analytics/models/intermediate/product/licenses/int_cloud_customers.sql

```diff
@@ -0,0 +1,20 @@
+-- Temporarily materialize during development
+{{
+    config({
+        "materialized": "table",
+        "snowflake_warehouse": "transform_l"
+    })
+}}
+
+select distinct a.installation_date
+    , a.installation_id as installation_id
+    , a.server_id as server_id
+    , b.customer_id as customer_id
+    , b.email as customer_email
+    , b.company_name as company_name
+    , coalesce(b.license_name, a.license_name) as license_name
+    , b.source as source
+from {{ ref('int_cloud_servers')}} a 
+left join {{ ref('int_cloud_licenses')}} b 
+on a.installation_id = b.installation_id
+where company_name is not null
```

Comment:
what happens if (or could it happen that) the `customer_id` is not null but the `company_name` is null or vice-versa?

Reply:
with no `company_name` this wouldn't be valuable information that we could use for this ticket. `customer_id` by itself isn't valuable with no other metadata tied to it.

Reply:
Added `email` check also. `customer_id` isn't going to be surfaced anywhere, its only used to compare or join with other sources.